packages/hierarchy-builder/src/hierarchy-builder/Utils.ts

```diff
@@ -3,8 +3,31 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
+import { ECClass, IMetadataProvider } from "./ECMetadata";
+import { getClass } from "./internal/Common";
+
 /**
  * This is an utility `Omit` type which works with union types.
  * @beta
  */
 export type OmitOverUnion<T, K extends PropertyKey> = T extends T ? Omit<T, K> : never;
+
+/** @beta */
```

Comment:
I believe it should be `@internal`

Reply:
Also, maybe it should be moved to `internal/Common.ts`?

Reply:
Moved to `internal/Common.ts` and made it `@internal`